src/cairo_interaction.rs

```diff
@@ -0,0 +1,81 @@
+// Copyright 2013-2015, The Rust-GNOME Project Developers.
+// See the COPYRIGHT file at the top-level directory of this distribution.
+// Licensed under the MIT license, see the LICENSE file or <http://opensource.org/licenses/MIT>
+
+//! Functions to support using cairo
+
+use std::mem;
+use glib::translate::*;
+use ffi::{self, GdkRGBA};
+use super::{Pixbuf, Window};
+use cairo::{Context, RectangleInt};
+
+pub fn create(window: &Window) -> Context {
```

Comment:
Since the module name isn't specific to `Context` those could go into similar traits for regions and surfaces it seems.